classes/controller/Controller.php

```diff
@@ -167,6 +174,14 @@ public function __construct()
             || strpos($_SERVER['HTTP_USER_AGENT'], 'Trident') !== false)) {
             header('X-UA-Compatible: IE=edge,chrome=1');
         }
+
+        /* _PS_CACHE_DIR_ */
```

Comment:
forget this comment, I put it here only to remember the constant name when I will store the container in cache: build a container is expensive in perf's, so we need to build it only once.